Code/Editor/EditorViewportWidget.cpp

```diff
@@ -742,7 +743,7 @@ void EditorViewportWidget::OnBeginPrepareRender()
     RenderAll();
 
     // Draw 2D helpers.
-    TransformationMatrices backupSceneMatrices;
+    //TransformationMatrices backupSceneMatrices;
```

Comment:
Remove?

Reply:
Wasn't sure, since that was used by previously commented out code ( the one below ) :smile: